.github/workflows/master-build.yaml

```diff
@@ -31,34 +31,26 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      # Create sha tagged image for container-agent
-      
-      # Deprecating the backend+spire image for now
-      # - name: Set backend-spire image name
-      #   run: echo "CONTAINER_BACKEND_WITH_SPIRE_TAG=ghcr.io/${{ github.repository_owner }}/tornjak-backend-spire-server:$GITHUB_SHA" >> $GITHUB_ENV
+     
+      # set DEV tags to have GITHUB_SHA  
       - name: Set github commit id
         run: echo "GITHUB_SHA=$GITHUB_SHA" >> $GITHUB_ENV
       - name: Set backend image name
-        run: echo "CONTAINER_BACKEND_TAG=ghcr.io/${{ github.repository_owner }}/tornjak-backend:$GITHUB_SHA" >> $GITHUB_ENV
+        run: echo "CONTAINER_BACKEND_DEV_TAG=ghcr.io/${{ github.repository_owner }}/tornjak-backend:$GITHUB_SHA" >> $GITHUB_ENV
```

Comment:
shouldn't they be `*_RELEASE_TAG`?

Reply:
no, the release tags are for the version, the dev tag is for the commit. 
the release tag is already set in the makefile
i see how this is confusing tho - we still have to push two image tags: one for with the version (currnetly RELEASE tag) and one with the git sha (currently the DEV tag as set here)

what would you propose we tag the latter if not "DEV"?